src/software/amazon/ion/IonInt.java

```diff
@@ -56,6 +56,15 @@ public long longValue()
     public BigInteger bigIntegerValue();
 
     /**
+     * Gets the actual type of the underlying content.
```

Comment:
What does "actual type" mean?  We've got two type systems at play here, Ion's and Java's.  So the docs need to be clear about which one is being discussed.